Formula/argocd-vault-plugin.rb

```diff
@@ -0,0 +1,34 @@
+class ArgocdVaultPlugin < Formula
+  desc "Argo CD plugin to retrieve secrets from Secret Management tools"
+  homepage "https://github.com/IBM/argocd-vault-plugin"
+  url "https://github.com/IBM/argocd-vault-plugin.git",
+    tag:      "v1.1.2",
+    revision: "b91d267dfcc69e8a0f029ca5afed585456f537a3"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+
+    datetime = `date -u +'%Y-%m-%dT%H:%M:%SZ'`
+
+    ldflags = %W[
+      -s -w
+      -X github.com/IBM/argocd-vault-plugin/version.Version=#{version}
+      -X github.com/IBM/argocd-vault-plugin/version.BuildDate=#{datetime}
+      -X github.com/IBM/argocd-vault-plugin/version.CommitSHA=#{Utils.git_head}
+    ].join(" ")
+
+    system "go", "build", *std_go_args, "-ldflags", ldflags, "."
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags), "."
```
Let's use the newer syntax for this.

Reply:
Actually, is the `"."` needed? Isn't that the default?

Reply:
Good call, it should not be needed